app/src/main/java/com/mealsmadeeasy/ui/home/profile/ProfileFragment.kt

```diff
@@ -0,0 +1,148 @@
+package com.mealsmadeeasy.ui.home.profile
+
+import android.app.AlertDialog
+import android.app.DatePickerDialog
+import android.content.Context
+import android.os.Bundle
+import android.support.design.widget.TextInputEditText
+import android.support.design.widget.TextInputLayout
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import com.mealsmadeeasy.R
+import com.mealsmadeeasy.ui.BaseFragment
+
+import java.util.*
+import android.widget.ArrayAdapter
+import com.mealsmadeeasy.model.Gender
+import com.mealsmadeeasy.model.UserProfile
+import java.text.SimpleDateFormat
+
+
+const val MIN_AGE = 13
+const val INCHES_IN_FEET = 12
+const val MIN_BMI = 15
+const val MAX_BMI = 50
+
+class ProfileFragment : BaseFragment() {
+
+    companion object {
+        fun newInstance(): ProfileFragment = ProfileFragment()
+
+        fun heightToInches(feet: Int, inches: Int): Int {
+            return feet * INCHES_IN_FEET + inches
+        }
```

Comment:
I think in Kotlin it's customary to not fret over static v. non-static methods (unless it absolutely _has_ to be static like the `newInstance` method here). So even though in Java this'd be static, I think it's more common to see it as a regular method in Kotlin.